lib/changesreader.js

```diff
@@ -165,11 +167,17 @@ class ChangesReader {
           if (self.fastChanges) {
             req.qs.seq_interval = self.batchSize
           }
-          if (self.selector) {
+          if (self.selector || self.fields) {
             req.qs.filter = '_selector'
-            req.body.selector = self.selector
+            if (self.selector) {
+              req.body.selector = self.selector
+            }
+            if (self.fields) {
```

Comment:
I don't see anything in the CouchDB docs saying that a `fields` array can be supplied. Am I missing something?